Sources/Commands/UserToolchain.swift

```diff
@@ -139,6 +145,8 @@ public struct UserToolchain: Toolchain {
             throw Error.invalidToolchain(problem: "could not find `clang` at expected path \(clangCompiler.asString)")
         }
 
+        self.xctest = try? AbsolutePath(Process.checkNonZeroExit(arguments: findArgs(utility: "xctest")).chomp())
```

Comment:
This can potential crash if we don't find `xctest`. We can use [this method](https://github.com/apple/swift-package-manager/blob/master/Sources/Basic/Path.swift#L507) to check if the string is a valid `AbsolutePath`.

Reply:
Please suggest a change?

Reply:
Check out https://github.com/apple/swift-package-manager/pull/1536/commits/db5bceeaaff637f2765c19bcdc8669cb49e328eb